config/jobs/kubernetes/test-infra/test-infra-periodics.yaml

```diff
@@ -36,7 +36,7 @@ periodics:
     base_ref: master
   spec:
     containers:
-    - image: gcr.io/k8s-prow/checkconfig:v20201006-e090651f0c
+    - image: gcr.io/k8s-prow/checkconfig:v20201008-84f802ee2a
```

Comment:
Don't be alarmed that this checkconfig job was at a different Prow version. This isn't due to any problem with the bump logic, but rather was caused by me adding this job without updating to the latest version before merging.